### PR TITLE
[common] fix creation of pidfile

### PR DESCRIPTION
**Pull request by DjLegolas**
https://dev.deluge-torrent.org/ticket/3278

### CODE REVIEW

File: deluge/argparserbase.py [hunk 1]
```diff
@@ -326,7 +326,7 @@ def _handle_ui_options(self, options):
             # Write pid file before chuid
             if options.pidfile:
                 with open(options.pidfile, 'wb') as _file:
-                    _file.write('%d\n' % os.getpid())
+                    _file.write(b'%d\n' % os.getpid())
```
Comment (cas--) [inline deluge/argparserbase.py]: Actually the pidfile should be opened in text mode so revert this and change `wb` to `w` above

**Comment by cas--** [comment]
The commit title should relate to what was fixed not how it was fixed. i.e. fix creating a pidfile

https://dev.deluge-torrent.org/wiki/GitTips#CommitMessages

**Comment by DjLegolas** [comment]
Fixed both commit message and code.

**Comment by cas--** [comment]
Thanks merged: 1b4ac88

**Pull request closed without merging**